app/Commands/SchedulingCommand.php

```diff
@@ -126,7 +126,7 @@ private function getSystemTimeZone(&$exitCode): string
                 $this->line('Please enter your "sudo" password so we can retrieve your timezone:');
                 return ltrim(exec('sudo systemsetup -gettimezone', $_, $exitCode), 'Time Zone: ');
             case Str::contains(php_uname('s'), 'Linux'):
-                return ltrim(exec('cat /etc/timezone'));
+                return exec('date +%Z', $_, $exitCode);
```

Comment:
Hey @jessarcher what if we check if the `/etc/timezone` file exists and use that when available (which should be the case in Debain-based distros)?
```suggestion
                if (file_exists('/etc/timezone')) {
                    return ltrim(exec('cat /etc/timezone', $_, $exitCode));
                }
                
                return exec('date +%Z', $_, $exitCode);
```